SECP-service/src/main/resources/assets/app/js/app.js

```diff
@@ -125,6 +125,17 @@ angular.module('SECP', ['ngResource', 'ngRoute', 'ui.bootstrap', 'ui.date',
               isAdmin: isAdmin
           }
       })
+      .when('/change-password', {
```

Comment:
you need to add these routes to the SECPService.java file

Reply:
For change password, its going to be tricky because I need the user-profile page directive first, so when that is approved to merge, then I can do that.

Reply:
Ok cool. 👍